src/styles/Input.module.css

```diff
@@ -1,7 +1 @@
-/* TODO: Might use MUI for all inputs for consistency */
-.default {
-  border-radius: 2px;
-  border: 1px solid var(--outline);
-  margin: 2px 4px;
-  padding: 4px 2px;
-}
+/* moved to Form.module.css since this was not styling input, but was styling the form */
```

Comment:
maybe remove this comment? it only makes sense if you know what used to be here.

Reply:
Will do. Thanks.